lib/matplotlib/artist.py

```diff
@@ -668,18 +668,16 @@ def set_clip_path(self, path, transform=None):
         """
         Set the artist's clip path, which may be:
 
-          * a :class:`~matplotlib.patches.Patch` (or subclass) instance
+        - a :class:`~matplotlib.patches.Patch` (or subclass) instance; or
+        - a :class:`~matplotlib.path.Path` instance, in which case a
+          :class:`~matplotlib.transforms.Transform` instance, which will be
+          applied to the path before using it for clipping, must be provided;
+          or
+        - ``None``, to remove the clipping path.
 
-          * a :class:`~matplotlib.path.Path` instance, in which case
-             an optional :class:`~matplotlib.transforms.Transform`
-             instance may be provided, which will be applied to the
-             path before using it for clipping.
-
-          * *None*, to remove the clipping path
-
-        For efficiency, if the path happens to be an axis-aligned
-        rectangle, this method will set the clipping box to the
-        corresponding rectangle and set the clipping path to *None*.
+        For efficiency, if the path happens to be an axis-aligned rectangle,
+        this method will set the clipping box to the corresponding rectangle
+        and set the clipping path to ``None``.
```

Comment:
Do we have any docs elsewhere that explains the clipping path/box distinction? If so, it would be nice to reference it here.